common/test/acceptance/accessibility/lms/test_account_settings_a11y.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
PR https://github.com/edx/edx-platform/pull/10081 just merged, so these should no longer go in the `common/test/acceptance/accessibility/` folder, but in with the other bok choy tests.  Sorry for the churn.  I'll send out a broader update too, but wanted to let you know here.  Thanks for writing these, they look great.